RevenueCat.xcodeproj/xcshareddata/xcschemes/StoreKitTests.xcscheme

```diff
@@ -71,15 +71,6 @@
       savedToolIdentifier = ""
       useCustomWorkingDirectory = "NO"
       debugDocumentVersioning = "YES">
-      <MacroExpansion>
-         <BuildableReference
-            BuildableIdentifier = "primary"
-            BlueprintIdentifier = "2DE20B7E26409EB7004C597D"
-            BuildableName = "StoreKitTestApp.app"
-            BlueprintName = "StoreKitTestApp"
-            ReferencedContainer = "container:Purchases.xcodeproj">
-         </BuildableReference>
-      </MacroExpansion>
```

Comment:
Is this intended?

Reply:
Yeah, I believe this is Xcode 13 cleaning up stuff when I refactored the framework name. It still works 🤷‍♂️